packages/kununu-utils/kununu-logger/index.js

```diff
@@ -48,16 +65,16 @@ export const customFormat = printf((info) => {
 });
 
 export const logger = createLogger({
+  levels: loggingLevels,
   format: format.combine(
     timestamp(),
     customFormat,
   ),
   transports: [
-    new (transports.Console)({
-      name: 'console',
-      colorize: true,
-      showLevel: true,
-      level: minimumLogLevel
+    new (FingersCrossed)({
+      level: 'debug',
```

Comment:
Without this line, logs below `info` level won't even enter on our custom transport... Winston will block it 🤔